main/ui/menu_ingame.rml

```diff
@@ -67,8 +67,8 @@
 
 							<h2> Bots </h2>
 							<indent>
-								<blocklink onClick='Events.pushevent("exec bot add Lavishment aliens", event)'> Add an alien bot </blocklink>
-								<blocklink onClick='Events.pushevent("exec bot add Pulsifer humans", event)'> Add a human bot </blocklink>
+								<blocklink onClick='Events.pushevent("exec bot add * aliens", event)'> Add an alien bot </blocklink>
```

Comment:
This might be overkill, but what about using lua to generate random bot names? Otherwise, just this change is sufficient to give the bots unique names.

Reply:
We can also not define default bot names at all, server owners will do it by the way.

Reply:
We can also name unnamed bots as `Bot#1` etc.

Reply:
I think this is acceptable honestly. We could also make a default set.

Reply:
btw, to answer the question, I think it would be overkill, or at least out of focus of this PR (same if we want to add the ability to set default bot prefix different than default player prefix).